test/extensions/one-tx-payment.js

```diff
@@ -111,9 +113,44 @@ contract("One transaction payments", accounts => {
 
       await fundColonyWithTokens(colony, token, INITIAL_FUNDING);
       await colony.moveFundsBetweenPots(1, 0, 0, d1.fundingPotId, d2.fundingPotId, WAD, token.address);
+      await oneTxExtension.makePaymentFundedFromDomain(1, 0, 1, 0, RECIPIENT, token.address, 10, 2, GLOBAL_SKILL_ID, { from: COLONY_ADMIN });
+    });
+
+    it("should allow a single-transaction to occur in a child domain, paid out from the root domain", async () => {
```

Comment:
In the interest of managing test times, I feel this test can be safely omitted -- I don't think it provides anything that the subsequent test (for domain 3) does not.

Reply:
I can go either way on this, but it does provide something in conjunction with the subsequent test - that the variable you correctly identified I had replaced with a constant that happened to work is a variable that changes correctly. We would fail mutant testing along those lines without both tests. Given the test is fast, I'm inclined to leave it in, but if I'm outvoted I'll remove it :smile:

Reply:
Ok, works for me.